samples/ballerina-by-example/examples/mutual-ssl/mutual-ssl.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina.net.http;
+import ballerina.net.http.response;
+import ballerina.lang.system;
+
+@http:configuration {
+    basePath:"/hello",
+    //Defines a https port.
```

Comment:
Should we group these structs i.e ssl.basePath etc ?

Reply:
We can not add structs inside annotations at the moment. This will be supported in the future